toto/verifylib.py

```diff
@@ -213,11 +213,15 @@ def verify_command_alignment(command, expected_command):
 
   for i in range(min(command_len, expected_command_len)):
     if command[i] != expected_command[i]:
-      raise Exception("Command '%s' and expected command '%s' do not align" \
-          % (command, expected_command))
+      raise CommandAlignmentFailed(
+          "Command '{0}' and expected command '{1}' do not align"
+          .format(command, expected_command))
   if command_len != expected_command_len:
-    log.warning("Command '%s' and expected command '%s' do not fully align" \
-        % (command, expected_command))
+    # FIXME we don't want print statements in a library
+    # The warning will be obsolete with a fix for
+    # https://github.com/in-toto/in-toto/issues/46
+    print("Command '{0}' and expected command '{1}' do not fully align"
```

Comment:
We figured that if in-toto is going to be used as a library, the calling application probably doesn't want to see messages from the library printed/logged to the console (at least not by default).  For applications that would like to have these log statements show up on the console, perhaps an option can be added for them?